Hestia/app/src/main/java/hestia/backend/RemoveDeviceTask.java

```diff
@@ -0,0 +1,82 @@
+package hestia.backend;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * This class will send a DELETE request to the Server in order to delete
+ * the Device "device".
+ */
+
+public class RemoveDeviceTask extends AsyncTask<Void, Void, Void> {
```

Comment:
We are already returning the response code in the deviceRetrieverTask. This decision was made so that we can handle the response code at a later point. It might occur that the device can not be deleted. In this case, we want to be able to notify the user. Therefore, we decided to return the HTTP response code. I agree with you that currently it is not useful, but it might be at some point. 

If you don' t add it here then we should remove it everywhere. This should happen only after discussing it with everyone on the client team, so that we can ensure that everyone is on the same page. For now, I think we should add it. We can always discuss it at a meeting.